app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -225,15 +221,13 @@ You will not be eligible for the international relocation payment if you are:
 
 ## How to apply for the international relocation payment (IRP)
 
-Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) are open between 2 April and 16 June 2024. 
+Applications for the [international relocation payment (IRP)](https://get-a-teacher-relocation-payment.education.gov.uk) are now closed. You can express your interest in the IRP by emailing us at IRP.ExpressInterest@education.gov.uk. We'll email you when applications reopen in autumn 2024.
```

Comment:
Suggest writing the acronym out for the link, just so anyone using a screen reader knows what it stands for if they're tabbing through links

Reply:
Hi @Sarah-DfE sorry not sure what you mean – I've got it spelt out AND as an acronym, do you mean just spell out?

Reply:
Sorry @laurahermionetennant92 - ignore me, I was getting confused!